src/main/java/core/basesyntax/Info.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Info {

```

Comment:
The name 'Info' as an interface is quite generic. Consider using a more descriptive name that indicates the kind of information this interface provides or the action it represents.